src/ontology/mondo-ingest.Makefile

```diff
@@ -524,9 +524,7 @@ $(REPORTDIR)/sync-subClassOf.direct-in-mondo-only.tsv: $(foreach n,$(ALL_COMPONE
 $(REPORTDIR)/sync-subClassOf.confirmed.tsv: $(foreach n,$(ALL_COMPONENT_IDS), $(REPORTDIR)/$(n).subclass.confirmed.robot.tsv)
 	awk '(NR == 1) || (NR == 2) || (FNR > 2)' $(REPORTDIR)/*.subclass.confirmed.robot.tsv > $@
 
-# todo: Until ODK has oaklib >= 0.5.20, need '$(MAKE) pip-oaklib' below https://github.com/INCATools/ontology-development-kit/issues/936
 $(REPORTDIR)/%.subclass.added.robot.tsv $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.direct-in-mondo-only.tsv $(REPORTDIR)/%.subclass.added-obsolete.robot.tsv: tmp/mondo-ingest.db tmp/mondo.db tmp/mondo.sssom.tsv
-	$(MAKE) pip-oaklib
```

Comment:
As the `todo` comment above says, this was only necessary so long as ODK's version of `oaklib` was < 0.5.20. This is now true for both the `dev` and `latest` containers.

```
docker run obolibrary/odkfull:latest pip freeze | grep oaklib
oaklib==0.5.25

docker run obolibrary/odkfull:dev pip freeze | grep oaklib
oaklib==0.6.5
sh run.sh pip freeze | grep oaklib
oaklib==0.6.5
```

I think this is one of the examples where this doesn't require a build before merging into `develop`. It can be double checked at `develop`->`main` pre-merge build time. But I will leave it up to you guys.